### PR TITLE
[DO NOT MERGE] RELEASE_ANNOUNCEMENTS.MD

### PR DESCRIPTION
**Pull request by kailuowang**
This PR is a draft of 1.0.0-MF announcement to publish in typelevel.org blog. It's not meant to be merged in cats, I created it merely to facilitate review.

### CODE REVIEW

File: RELEASE_ANNOUNCEMENTS.MD [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+* @raulraja
+* @RawToast
+* @sellout
+* @stew
+* @sullivan-
+* @SystemFw
+* @takayuky
+* @tel
+* @TomasMikula
+* @toplecat
```
Comment (peterneyens) [inline RELEASE_ANNOUNCEMENTS.MD]: I almost hoped there really was a @tpolecat copycat called @toplecat :smile:

Comment (tpolecat) [inline RELEASE_ANNOUNCEMENTS.MD]: pthbbbt!
image

File: RELEASE_ANNOUNCEMENTS.MD [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+**MF** stands for *milestone final*,
+this will be the last release before cats 1.0.0-RC1 which will be followed by cats 1.0 shortly. 
+
+The main purpose/focus of this release is to offer a relatively stable API to work with prior to the official 1.0.
+It can be deemed as a proposal for the final cats 1.0 API. Please help test it and report any improvements/fixes
+needed either in the [cats-dev gitter channel](https://gitter.im/typelevel/cats-dev) or as [github issues](https://github.com/typelevel/cats/issues/new). 
+Post cats 1.0, we will keep API stable and maintain strong binary compatibility. 
+
+Highlights of the major new features include but not limit to:
```
Comment (durban) [inline RELEASE_ANNOUNCEMENTS.MD]: "limited"?

File: RELEASE_ANNOUNCEMENTS.MD [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+There are more breaking changes in this release - we want to include as many necessary breaking changes as possible in this release 
+to reach stability. Please follow the [migration guide][migration guide] from 0.9.0 in the  [release notes][release notes]. 
+
+
+
+### What's next
+
+Although we made many improvements to the documentation in this release, it's still by and large a WIP.
+The next release 1.0.0-RC1 will focus documentation and API refinement based on community feedback. 
+RC1 is scheduled to release in September. Unless the amount of bug fixes warrants a RC2, it's likely that
```
Comment (durban) [inline RELEASE_ANNOUNCEMENTS.MD]: "be released"?

**Comment by kailuowang** [comment]
@durban thanks for the review! really appreciate it.

**Comment by tpolecat** [comment]
Have we bikeshedded **-MF** yet? If it's a proposed final API why don't we go ahead and call it **RC1**? Apologies if I missed a prior argument.

**Comment by kailuowang** [comment]
@tpolecat we did bike shed *MF* in this [issue](https://github.com/typelevel/cats/issues/1595) and gitter. The idea of this coming release is to put the API as a release out there for users to test and provide feedbacks so that we can finalize the API. I originally proposed RC1 as the name of this release, and you suggested that RC means "done", which I agreed. Thus I came up with the *MF*.  This release scheme is not a very common practice, but given that cats is a very community driven project, I think it's essential to collect more feedbacks before locking down to an 1.0 API we have to maintain bin compat. 
I understand that this release scheme isn't very common, that's why I drafted this blog post to intended to better communicate the design. It looks like this communication is still vague. English isn't my native language, so I am soliciting advice/suggestion/help here.

**Comment by tpolecat** [comment]
Gotcha, thanks, withdrawn. :-)

**Comment by kailuowang** [comment]
announcement made

**Pull request closed without merging**